### ISSUE TITLE
RAD-43 & 55 audit - fix unintended update to type for homeCommunityId

### ISSUE THREAD

**Issue by lynnfel** [IHE/publications#211]
One of the participants preparing for the September IHE Connectathon caught an error in CP-ITI-1136 (https://docs.google.com/document/d/11_rlHNdEH4oefmR2LWwUi-e7MoYHx-hE/edit?usp=sharing&ouid=105005564211348147894&rtpof=true&sd=true) which was balloted and approved in ITI Ballot 60, and then integrated into the ITI Technical Framework Final Text Rev 19.0 which was published in June 2022.

Read the rationale in the CP .   I think we intended NOT to change the audit message for ITI-43 and ITI-55, even though the value for “type” for homeCommunityId is inconsistent with other audit messages elsewhere in the TF.   In fact, we added a note acknowledging the inconsistency.   BUT…we mistakenly updated the type anyway.

This github issue fixes 'type' which should be valued with  “ihe:homeCommunityID” and not “urn:ihe:homeCommunityID” in order to not cause a breaking change.

**Comment by lynnfel** [comment]
The fixes to ITI-43 and ITI-55 audit messages are in https://github.com/IHE/publications/tree/issue-211-fix

**Comment by lynnfel** [comment]
https://github.com/IHE/publications/pull/212

**Comment by JohnMoehrke** [comment]
looking at the proposed change. I understand the ramifications much better.  (It was hard to follow with the word CP which had so much else in it)

If we do not revert this, thus accept a breaking change for the sake of consistency and proper use of urn:ihe prefix, then we should understand the ramifications of the break. I am not convinced that this ramifications of this breaking change are all that sustentative. Yes some older audit log entries will be using the old id while newer would would use the newer id.. but in both case the data are there.

However, as indicated this change seems to be understood as not intended, and intended to keep the non consistent home community id encoding. 

I don't have much of a preference. So given that a member using our specifications has indicated the concern, we should follow that member need. Thus accept the pull-request and revert to how this was indicated before.
